### DISCUSSION TITLE
Change math mode font

### DISCUSSION THREAD

**Discussion by DiogoCabanas** [joaomlourenco/novathesis#384] [Q&A] [answered]
Hello! First, let me thank you for the fantastic work you did in building this template @joaomlourenco. It has been an enormous help.
My question is how can I change the math mode font? It is slightly different from the "normal" one, when you do an article-type document on overleaf, for example.

Thanks in advance for your help!
Best regards,
Diogo Cabanas

**Comment by joaomlourenco** [comment, marked as answer]
@DiogoCabanas 
There are multiple _font sets_ defined in the NOVAthesis template, and you can switch between them using the command
```latex
 \ntsetup{fontstyle=<SOMETHING>}
```
in `Config/1_novathesis.tex`.  Maybe you'll be happier with one of the alternatives.

You may even create your own font style…  The definition for these font styles are in the folder "NOVAthesisFiles/FontStyles/".   Jut create a file and give it a name, e.g.,  "myfontstyle.ldf", and then use ` \ntsetup{fontstyle=myfontstyle}`.  See the examples for inspiration.  ;)

**Reply by DiogoCabanas** [reply]
Thanks for the quick and helpful reply. My question was more in the sense of trying to figure out how to change only the font in the math mode. I will try to work it out by changing the font styles.

Thank you again.
